### DISCUSSION TITLE
share local resources(分享本地包-js源)

### DISCUSSION THREAD

**Discussion by hjdhnx** [Hiram-Wong/ZyPlayer#180] [Show and tell]
如题，新版zyplayer从4月27号开始支持了本地服务，可以把t3类型的js源和配置做成本地包，让zyplayer能够通过基础配置一键导入hipy源获取。大家可以制作并分享好用的本地包了

**Comment by hjdhnx** [comment]
你这里做个示范，本地包收集自高天流云仓库
[drpy_gao.zip](https://github.com/Hiram-Wong/ZyPlayer/files/15140388/drpy_gao.zip)

**Reply by kingshiplet** [reply]
现在2025.4月。请问所有的资源都要挂梯子才能看是怎么回事？

**Comment by z1298808165** [comment]
大佬，请问下本地宝怎么使用呐，没找到导入的地方，是需要手动放入某个文件夹吗

**Comment by Hiram-Wong** [comment]
> 大佬，请问下本地宝怎么使用呐，没找到导入的地方，是需要手动放入某个文件夹吗

文档https://zy.catni.cn/edit-source.html#%E6%9C%8D%E5%8A%A1

**Reply by bobobella** [reply]
打不开文档，请求被拒绝

**Comment by hjdhnx** [comment]
2024/05/05 本地包分享（同步最新hipy项目js源）
注意:此版本不带电视直播配置，带直播的本地包需要进q群下载
[drpy_dz.zip](https://github.com/Hiram-Wong/ZyPlayer/files/15213102/drpy_dz.zip)

**Reply by flyinghmx** [reply]
是在基础管理-数据管-一键配置中，选择hipy，导入吗？没显示成功，求指导。

**Reply by flyinghmx** [reply]
Q群在哪里？

**Comment by q7954** [comment]
道长我想问下，国内云服务器能不能搭建hipy？还有cms采集能搭建自用吗？这个问题如果这里不方便回答的话能不能在群里说下

**Reply by Hiram-Wong** [reply]
搭建肯定是可以玩的，就是不合法罢了，自用到影响也不大

**Reply by q7954** [reply]
只要不分享出去，不会上门查水表对吧

**Reply by Hiram-Wong** [reply]
那我只能说不好说，一般大厂的机器差的严一点

**Comment by Mubai-meng** [comment]
> 2024/05/05 本地包分享（同步最新hipy项目js源）
> 注意:此版本不带电视直播配置，带直播的本地包需要进q群下载
> [drpy_dz.zip](https://github.com/Hiram-Wong/ZyPlayer/files/15213102/drpy_dz.zip)
> 

哥，求组织（群号）

**Comment by hjdhnx** [comment]
[drpy_dzlive_0518.zip](https://github.com/Hiram-Wong/ZyPlayer/files/15360490/drpy_dzlive_0518.zip)

**Reply by hjdhnx** [reply]
最新zyplayer可用本地包。同时支持手机端easybox

**Reply by xintiere** [reply]
zyplayer3.5.5 0509-3   大佬，最新的不是3.3.6吗？这个3.5.5是啥。。。

**Reply by vankeoa** [reply]
可以更新一下这个本地包吗，可以适用最新的3.4.1以上的数据结构吗？

**Reply by vankeoa** [reply]
大佬希望您能看到，帮小白一下。

**Comment by polar1114** [comment]
有没有分享源的组织

**Comment by Misonnnn** [comment]
求群号和组织。
现在直播看不了。

**Reply by Hiram-Wong** [reply]
直播最近严查哦

**Comment by Misonnnn** [comment]
是不是有什么群组织啊，求入群账号






| |
Mison的邮件
|
|
***@***.***
|
---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年8月4日 13:25 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***>,
***@***.***> |
| 主题 | Re: [Hiram-Wong/ZyPlayer] share local resources(分享本地包-js源) (Discussion #180) |

直播最近严查哦

**Comment by keneece** [comment]
求群号和组织~

**Comment by bobobella** [comment]
求助，为什么导进去全是”哎呀出了点差错“
<img width="583" alt="Screenshot 2025-03-15 at 23 07 17" src="https://github.com/user-attachments/assets/99e68976-732c-4c24-9251-39e46d30d318" />

**Reply by Hiram-Wong** [reply]
类型全空的，说明导入方法不对

**Comment by S823790653** [comment]
源动力，导入显示：“失败:AxiosError: Request failed with status code 500”，是啥原因

**Reply by Hiram-Wong** [reply]
1.ip 没授权或者 ip 变了
2.链接错误，重复粘贴，自己观察输入框
3.关vpn 重新授权
4.尝试增加全局超时时间

**Reply by S823790653** [reply]
感谢，发现原因了，粘贴时不知为何粘贴了两次。已经解决

**Reply by 1WeiLiang1** [reply]
3.3.10版本配置有吗

**Comment by 1WeiLiang1** [comment]
3.3.10版本配置有吗

**Comment by keneece** [comment]
没有呢，好久没有了，只有3.1.10的
&nbsp;


串串儿
***@***.***



&nbsp;

**Comment by liuyibing316-blip** [comment]
新版不会配置
